error[E0499]: cannot borrow `test` as mutable more than once at a time
  --> tests/compile_fail/from_fn_mut_borrow_self_twice.rs:15:17
   |
14 |     let init1 = test.test();
   |                 ---- first mutable borrow occurs here
15 |     let init2 = test.test(); // fails
   |                 ^^^^ second mutable borrow occurs here
16 |     drop((init1, init2));
   |           ----- first borrow later used here
